ivre/db/mongo.py

```diff
@@ -182,6 +182,12 @@ def migrate_schema(self, colname, version):
             updated = False
             new_version, migration_function = self.schema_migrations[
                 colname][version]
+            if config.DEBUG:
```

Comment:
The `logging` module seems more adapted for this (but it will certainly need a bigger patch).